### ISSUE TITLE
Discussion Forum improvement

### ISSUE THREAD

**Issue by rajatgupta24** [codeforcauseorg/edu-client#333]
## Is your feature request related to a problem? Please describe.
hey @Abhishek-kumar09, I'd like to work on the discussion forum as right now it is more like a static single page which might make it hard to use. 

## Describe the solution you'd like
- I'd like to make different sections for all discussions & my discussions (in which I took part) to make it more classified. 
- I'd like to create tags to make it easier to search discussions related to the same topics.

**Comment by rajatgupta24** [comment]
Hey @Abhishek-kumar09, I'd like to make these changes.

**Comment by Abhishek-kumar09** [comment]
Yes, we are creating design for this page, Will update you soon. I think we can have a group meeting to gather some thoughts from you and everyody else.

**Comment by rajatgupta24** [comment]
> Yes, we are creating design for this page, Will update you soon. I think we can have a group meeting to gather some thoughts from you and everyody else.

Good, I've a basic outline im mind, it'd be great to discuss.
